parachain/pallets/polkaeth-app/src/lib.rs

```diff
@@ -6,57 +6,44 @@ use frame_system::{self as system, ensure_signed};
 use frame_support::{
 	decl_error, decl_event, decl_module, decl_storage,
 	dispatch::{DispatchResult, DispatchError},
-	traits::{Currency, ExistenceRequirement, WithdrawReason, WithdrawReasons},
 };
 use sp_std::prelude::*;
-use artemis_core::{AppID, Application, Message};
-use codec::Decode;
+use sp_core::{H160, U256, RuntimeDebug};
 
-use sp_std::convert::TryInto;
+use artemis_core::{AppID, Application, RelayEventEmitter, Message};
+use codec::{Encode, Decode};
 
 use artemis_ethereum::{self as ethereum, SignedMessage};
+use artemis_generic_asset as generic_asset;
 
 #[cfg(test)]
 mod mock;
 
 #[cfg(test)]
 mod tests;
 
-pub type PolkaETH<T> =
-	<<T as Trait>::Currency as Currency<<T as system::Trait>::AccountId>>::Balance;
+pub const APP_ID: &[u8; 32] = &[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
As above

Reply:
Looking to tackle this in M5.

Reply:
@vgeddes please add github issue or trello task